src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -26,25 +26,29 @@ Connect Sentry to your AWS account to automatically add Sentry error and perform
 
    ![](project.png)
 
-1. Click on **Add Sentry's CloudFormation to your AWS**.
+1. Click on **Go To AWS**.
 
-   ![](aws_account.png)
+   ![](aws_account_1.png)
 
 1. Scroll to the bottom of the Create Stack page, check the box, then press **Create Stack** to create a CloudFormation stack in your current region. This stack allows Sentry to automatically instrument your Lambda functions.
 
    ![](cloudformation.png)
 
-1. Wait for the stack to be created, which is confirmed by seeing the status change to **Create Complete**, then copy your AWS account number, select the region you want to use, and press **Next**.
+1. Wait for the stack to be created, which is confirmed by seeing the status change to **Create Complete**, then click **I've created the stack**.
+
+   ![](aws_account_2.png)
+
+1. Now copy your AWS account number, select the region you want to use, and press **Next**.
 
    ![](stack_created.png)
 
 <Alert title="Note" level="info">
 
-You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+You need to create the CloudFormation stack only once per account, even when installing the integration in multiple regions. If you are setting up the integration in a second region, copy the externalId from the existing CloudFormation stack on the "Parameters" tab into the form shown above.
 
 </Alert>
 
-1. Select the Lambda functions to automatically add the Sentry layer and necessary environment variables when you **Finish Setup**. **Note:** You can enable or disable Sentry on Lambda functions after installation.
+8. Select the Lambda functions to automatically add the Sentry layer and necessary environment variables when you **Finish Setup**. **Note:** You can enable or disable Sentry on Lambda functions after installation.
```

Comment:
I don't think this markdown is formatting the number correctly
<img width="417" alt="image" src="https://user-images.githubusercontent.com/4205004/108444923-84131200-7229-11eb-9031-87ab942c8be1.png">

Reply:
Is that from before? Cause this is what I see in the preview build:

![Screen Shot 2021-02-18 at 5 41 23 PM](https://user-images.githubusercontent.com/8533851/108445015-89179780-7210-11eb-84ac-fcc1c14dff67.png)

Reply:
Nevermind, yup. TIL the preview's search switches to docs.sentry.io

Reply:
Yes, that can happen :( - you think you're looking at the preview, when in fact you're in live docs. Only way to confirm what you're seeing is by viewing the URL.